### ISSUE TITLE
Commodore CPM Cartridge does not work with U64, v1.06 3.3 

### ISSUE THREAD

**Issue by xlar54** [GideonZ/1541ultimate#69]
Obviously low on a priority list for such exotic and unwanted hardware, but am creating a ticket for you incase its compatibility helps improve other things as well.  

The cpm disk starts to load, then the machine resets.

**Comment by GideonZ** [comment]
1.06 is outdated. It was superseded with 1.0B and later 1.10. Please re-open the ticket if it still does not work with 1.18, which will be released shortly.

**Comment by xlar54** [comment]
Ok, FYI, I tried with 1.10.  It definitely went further than 1.06.  The asterisks that get displayed while CPM is booting does display.  Once they get to where they should, the screen goes to a light blue or cyan blank.  Disk light stays on.  I will test again after 1.18 as well.  (Btw, REU was disabled, command interface was disabled, no roms or jiffydos, etc... as vanilla as I could get it)

Do note that I tried the cart on a C128D (I dont have an ordinary C64), and it booted up ok to CPM using the same disk, however I got odd characters on the screen and wasnt very usable.  This is not normal for my 128d, so its possible / likely the CPM cart or Z80 inside it is faulty.   Or... the 128 just doesnt like that cart.  Hard to know for sure.  But will see what happens with 1.18.
